internal/integration/provision/upgrade.go

```diff
@@ -70,48 +70,49 @@ type upgradeSpec struct {
 }
 
 const (
-	previousRelease = "v0.8.4"
-	stableRelease   = "v0.9.0-alpha.5"
+	stableRelease = "v0.8.5"
+	devRelease    = "v0.9.0-beta.2"
```

Comment:
could we please keep it the way it is, i.e. `stableRelease`?

we'll update v0.9.0-beta.2 to 0.9.0 once we have it released, but it will stay as stable

Reply:
It is either wrong now (alpha and beta releases are not stable, and that's very confusing) or later.

Once 0.9.0 is out, will we support 0.8.x?

Reply:
bumping version is good of course, my comment was only about naming

Reply:
I don't particularly like the word "stable" there because alphas and betas are not stable.
I would call it "current", but that word is already used for the version being built.
"next" is not good too.
What about "latest"?

Reply:
Ended up with `stableRelease` and a comment